commands/pass.go

```diff
@@ -0,0 +1,59 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/bndw/pick/utils/pswdgen"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+func init() {
+	cmd := &cobra.Command{
+		Use:   "pass",
+		Short: "Generate a password without storing it",
+		Long:  "The pass command is used to generate a password without storing it.",
+		Run: func(cmd *cobra.Command, args []string) {
+			runCommand(Pass, cmd, args)
+		},
+	}
+	cmd.Flags().Int("length", pswdgen.DefaultPasswordLength, "the length of the generated password")
```

Comment:
I agree the introduction of an `--interactive` flag affect other flags. How about this- let's ship this feature as is. If the interactive bit becomes a painpoint in my usage, I'll open an issue and we can revisit the defaults/controls for the `pass` command. :)